src/controllers/formExampleController.ts

```diff
@@ -22,14 +23,15 @@ export class FormExampleController {
 
   public async post(req, res, next) {
     let formExampleModel = new FormExampleModel(
-      req.body.fullName,
+      lettersAndSpaceInputString(req.body.fullName),
       parseInt(req.body.dobDay, 10),
       parseInt(req.body.dobMonth, 10),
       parseInt(req.body.dobYear, 10),
       req.body.preferredContactOption,
-      req.body.contactEmail,
+      normalizeEmailInput(req.body.contactEmail),
       req.body.contactPhone,
-      req.body.contactSmsNumber
+      req.body.contactSmsNumber,
```

Comment:
Could we use isMobilePhone from validator package to check this field?

Reply:
done